zephyr/lib/alloc.c

```diff
@@ -47,27 +47,9 @@ extern struct tr_ctx zephyr_tr;
  * Currently functional but some items still WIP.
  */
 
-#ifndef HEAP_RUNTIME_SIZE
-#define HEAP_RUNTIME_SIZE	0
-#endif
-
-/* system size not declared on some platforms */
-#ifndef HEAP_SYSTEM_SIZE
-#define HEAP_SYSTEM_SIZE	0
-#endif
-
 /* The Zephyr heap */
```

Comment:
Hmm, sure it's ok to just remove them?

Reply:
ok, leaving as it was, does not hurt anyway